CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.8.0 (TBD)
+## 1.8.0 (2023-09-14)
 
 ### New Features
 
```

Comment:
there's an extra blank line at line 13
extra space before period at line 12
line 26: add period at end of sentence